salt/modules/mongodb.py

```diff
@@ -156,6 +156,36 @@ def db_remove(name, user=None, password=None, host=None, port=None, authdb=None)
     return True
 
 
+def _version(mdb):
+    return mdb.command('buildInfo')['version']
+
+
+def version(user=None, password=None, host=None, port=None, database='admin', authdb=None):
+    '''
+    Get MongoDB instance version
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' mongodb.version <user> <password> <host> <port> <database>
+    '''
+    conn = _connect(user, password, host, port, authdb=authdb)
+    if not conn:
+        return 'Failed to connect to mongo database'
```

Comment:
Maybe it would be good to log this as an error instead of just returning the string?